### PR TITLE
Change the model ID to new Default Model

### PR DESCRIPTION
**Pull request by Cing-Chen**
According to the latest ChatGPT website, they already optimized a new model which ID is "text-davinci-002-render-sha".
"text-davinci-002-render-sha" is the original Turbo Model, and becomes the Default Model now.

image
The link of this model is `https://chat.openai.com/chat?model=text-davinci-002-render-sha`.

### CODE REVIEW

File: src/revChatGPT/V1.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def ask(
             "parent_message_id": parent_id or str(uuid.uuid4()),
             "model": "text-davinci-002-render-sha"
             if not self.config.get("paid")
-            else "text-davinci-002-render-paid",
+            else "text-davinci-002-render-sha",
```
Comment (acheong08) [inline src/revChatGPT/V1.py:147]: sha is already the default. Paid is for if the user is "plus"

**Review by acheong08 — COMMENTED** [review]
Unless there is some info I'm unaware of, this simply forces paid users to use a now free model

**Comment by Jecket22** [comment]
They are actually not wrong - ChatGPT Plus offers two models: Default (`text-davinci-002-render-sha`, formerly "Turbo") and Legacy (`text-davinci-002-render-paid`).
Interestingly, the current free model also happens to be the same one for the paid one (`text-davinci-002-render-sha`)!
So I think one can make a model switcher for paid users instead of   leaving a redundant if statement for choosing the same model

**Comment by Cing-Chen** [comment]
Got it, sorry for the wrong PR QQ
I will be careful to PR in furure.
Thank you all!

**Pull request closed without merging**